contracts/SplitEquippable/README.md

```diff
@@ -0,0 +1,1348 @@
+    [
+      BigNumber { value: "7" },
+      1,
+      'ipfs://wings/1.svg',
+      partId: BigNumber { value: "7" },
+      z: 1,
+      metadataURI: 'ipfs://wings/1.svg'
+    ]
+  ],
+  slotParts: [
+    [
+      BigNumber { value: "9" },
+      BigNumber { value: "2" },
+      4,
+      BigNumber { value: "1" },
+      '0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9',
+      '',
+      partId: BigNumber { value: "9" },
+      childResourceId: BigNumber { value: "2" },
+      z: 4,
+      childTokenId: BigNumber { value: "1" },
+      childAddress: '0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9',
+      metadataURI: ''
+    ],
+    [
+      BigNumber { value: "10" },
+      BigNumber { value: "3" },
+      4,
+      BigNumber { value: "2" },
+      '0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9',
+      '',
+      partId: BigNumber { value: "10" },
+      childResourceId: BigNumber { value: "3" },
+      z: 4,
+      childTokenId: BigNumber { value: "2" },
+      childAddress: '0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9',
+      metadataURI: ''
+    ],
+    [
+      BigNumber { value: "11" },
+      BigNumber { value: "8" },
+      4,
+      BigNumber { value: "3" },
+      '0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9',
+      '',
+      partId: BigNumber { value: "11" },
+      childResourceId: BigNumber { value: "8" },
+      z: 4,
+      childTokenId: BigNumber { value: "3" },
+      childAddress: '0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9',
+      metadataURI: ''
+    ]
+  ]
+]
+````
+
+This concludes our work on the simple Split equippable RMRK lego composite and we can now move on to examining the
+advanced implementation.
+
+## Advanced SplitEqiuppable
+
+The advanced `SplitEquippable` consists of three smart contracts. The
+[`AdvancedBase`](../MergedEquippable/README.md#advancedbase) is already examined in the `MergedEquippable`
+documentation. Let's first examine the `AdvancedExternalEquip` and then move on to the `AdvancedNestingExternalEquip`.
+
+**NOTE: As the `AdvancedBase` smart contract is used by both `MergedEquippable` as well as `SplitEquippable` it resides
+in the root `contracts/` directory.**
+
+### AdvancedExternalEquip
+
+The [`AdvancedExternalEquip.sol`](./AdvancedExternalEquip.sol) smart contract represents the minimum required
+implementation in order for the smart contract to be compatible with the `MultiResource` and `Equippable` part of the
+`ExternalEquip` RMRK lego composite. It uses the
+[`RMRKExternalEquip`](https://github.com/rmrk-team/evm/blob/dev/contracts/RMRK/equippable/RMRKExternalEquip.sol) import
+to gain access to the `MultiResource` and `Equippable` part of the External equippable RMRK lego composite:
+
+````solidity
+import "@rmrk-team/evm-contracts/contracts/RMRK/equippable/RMRKExternalEquip.sol";
+````
+
+We only need the `nestingAddress`, which is the address of the deployed `AdvancedNestingExternalEquip` smart contract,
+in order to properly initialize it after the `AdvancedExternalEquip` inherits it:
```

Comment:
Might be worth nothing that we could use address(0) to initialize the contract and later set the nesting address. So you can deploy either contract first.

Reply:
Good point 🔥 

I'll add a note about this 👌